### PR TITLE
Enhance "Verify That XXXXXX Image Is A CPaaS Built Image" tests

### PR DESCRIPTION
**Pull request by jgarciao**
Tests were failing sometimes because prometheus pod wasn't running yet after installation. Adding a wait until the prometheus pod is running

### CODE REVIEW

**Review by asanzgom — DISMISSED** [review]

**Review by jstourac — APPROVED** [review]

File: ods_ci/tests/Tests/100__deploy/100__installation/102__post_install.robot [hunk 1]
```diff
@@ -123,6 +118,9 @@ Verify Oath-Proxy Image Is A CPaaS Built Image
     [Tags]      Sanity
     ...         Tier1
     ...         ODS-666
+    Skip If RHODS Is Self-Managed
+    Wait For Pods To Be Ready    label_selector=deployment=prometheus
+    ...    namespace=${MONITORING_NAMESPACE}    timeout=60s
```
Comment (bdattoma) [inline ods_ci/tests/Tests/100__deploy/100__installation/102__post_install.robot:123]: this refers to Monitoring NS, but the next lines refers to dashboard pods. Is it expected?

Comment (jgarciao) [reply ods_ci/tests/Tests/100__deploy/100__installation/102__post_install.robot:123]: You are right. I thought it was the oauth-proxy for prometheus, but as you saw is for dashboard. Fixing it

Comment (jgarciao) [reply ods_ci/tests/Tests/100__deploy/100__installation/102__post_install.robot]: Fixed now

**Review by bdattoma — APPROVED** [review]

**Comment by jgarciao** [comment]
Tested successfully in local and rhods-ci-pr-test/2351 (only oauth-proxy test, as the others were skipped)

**Review by jstourac — APPROVED** [review]

**Pull request merged**